### ISSUE TITLE
"Report..." links in the "Feedback" menu at the top of MDN pages is confusing

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/yari#1435]
The "Report a content problem" and "Report an issue" links in the "Feedback" menu are confusing - it is not clear what the difference is. Better would be something like "Report a content problem" and "Report a platform problem" that just goes to the generic new issue page on the relevant repos.

**Comment by chrisdavidmills** [comment]
> How am I - as user - supposed to know, whether something is a content or platform problem?

This is a good point. We'd have to play with the wording a bit to see what is the most inuitive. Maybe "Report a content problem" and "Report a problem with the site itself". We are never going to make it completely foolproof of course...

**Comment by Ryuno-Ki** [comment]
That sounds easier to understand to me.

Perhaps this should be moved to a step in the process after the problem was reported? (a.k.a. triaging)
I mean, it's more important to make people inform you about something than making them put it in the right basket, isn't it?

**Comment by peterbe** [comment]
How about a "Report a site problem" or "Report a general problem"?

Whilst we ponder that, I'm going to go ahead and work on the change itself and we can tune the "prose" when I have a prototype.

**Comment by mattiapontonio** [comment]
How about: 

- YARI's issues
- Contents's issues

?

I like the _Issues_ word because is how is called GitHub.

A issue is not necesseraly a problem.

That YARI Is the Front End project Is something people have to understand sometime.
Content is content.

### side issue
There Is a way to auto Copy the URL in the issue (in my case in the Android App)

**Comment by peterbe** [comment]
@chrisdavidmills I tried to work on this and it's still really hard. There are technical challenges too. E.g. if you're on the home page or sign-up page or site-search or Page-not-found which don't have a GitHub URL thing like we do in the document footer. And if you use client-side navigation, it's non-trivial how you change the link in the main menu since that's a parent component that doesn't know what's going on deeper in the React tree. 

But the problem still stands, like @Ryuno-Ki brought up:

> How am I - as user - supposed to know, whether something is a content or platform problem?

It's really a good point. What's a platform? Is the article text content or is it for reporting spam or inappropriate stuff?

The link down in the document footer is so much better. We should definitely get people to use that to report content (we here know what "content" means now) issues. In fact, if we one day wanna ask or append questions depending on dynamic stuff, that's the place to do it. Or perhaps that's the place where we can list active open issues relating to the current URL. 

So part of me wishes we could just make the "Report a content problem" be an anchor link to the bottom of the page. But then I'm not sure what it's supposed to do if you're not on a document page. 

And somewhere there really should be a link to report general platform (Yari) issues. 

To avoid this becoming a timesink, I'm going to adopt a very simple solution.

1. There are two links now: ["Report a content issue"](https://github.com/mdn/content/issues/new) and ["Report a platform issue"](https://github.com/mdn/yari/issues/new). Note the use of the word "issue" instead of "problem".
2. If you're on a document page that has a `#on-github` section at the bottom of the page, it scrolls down to that instead. 
3. If you're *not* on a document page, it proceeds to go to https://github.com/mdn/content/issues/new

It's a bit weird and it's even a bit hacky. But I think it'll unblock us to proceed and potentially improve this once we've launched. 
Part of me wants to delete the links entirely but the top menu item is called "Feedback" and that makes sense to have up there. 

Would appreciate your feelings on this too @escattone and @fiji-flo 
@schalkneethling I'll send you the PR but obviously, very much appreciate your feelings on the above-mentioned too.

**Comment by mattiapontonio** [comment]
It occurred to me that if you can pass the URL to GitHub maybe you can pass the user agent as well

**Comment by schalkneethling** [comment]
@peterbe @chrisdavidmills Looking at the content of the "Feedback" top-level menu:

![Screenshot of expanded feedback top-level menu](https://user-images.githubusercontent.com/10350960/101353318-0e3f1180-389c-11eb-8b69-ea3269973104.png)

I wonder if we should not consider removing it entirely. The entries in there are:

- Send feedback (Already a duplicate link in the site footer)
- Get Firefox help (this could be a link in the site footer above the link to Firefox)
- Get Web Development help (links to Stackoverflow - Does anybody ever click this? Btw. we have amazing web development help one top-level menu to the left 😄)
- Join the community (perhaps replace Feedback with just this link at the top level. This seems in line with our renewed focus on community i.e. have this exposes at the top level)
- Report a content problem (Taken care of by the new Github section)
- Report an issue (Taken care of by the new Github section)

![Screenshot 2020-12-07 at 15 00 14](https://user-images.githubusercontent.com/10350960/101353819-ec925a00-389c-11eb-903c-9e35f215a20a.png)


Apologies for not having mentioned this earlier 😢 ~ Thoughts?

**Comment by chrisdavidmills** [comment]
I think you've got a good point here. I'd be happy to try your suggestion and then see what we think.

Have you got any analytics on how many times these links have been clicked?

**Comment by schalkneethling** [comment]
> I think you've got a good point here. I'd be happy to try your suggestion and then see what we think.
> 
> Have you got any analytics on how many times these links have been clicked?

We do send pings but, you would have to group it by url to knw the exact numbers as they are all just menu clicks. Let me see if I can get a rough idea.

**Comment by schalkneethling** [comment]
@chrisdavidmills 

This is for the last month

### Feedback

The feedback link(which I assume might be what people click to report a problem, maybe?) is clicked most of all:

Unique: 13,387(2.51%)
Total: 20,268(2.68%)

### Stackoverflow:

Unique: 1,133(0.21%)
Total: 1,908(0.25%)

### Sumo:

Unique: 1,467(0.28%)
Total: 1,856(0.25%)

### Community

Unique: 1,080(0.20%)
Total: 2,135(0.28%)

### Issues

Specifically https://github.com/mdn/kuma/issues/new/choose

Unique: 639(0.12%)
Total: 1,191(0.16%)

Specifically https://github.com/mdn/sprints/issues/new?template=issue-template.md&projects=mdn/sprints/2&labels=user-report&title=https://developer.mozilla.org/en-US/

Unique: 37(0.01%)
Total: 699(0.09%)

### URL

https://analytics.google.com/analytics/web/?authuser=1#/report/content-event-events/a36116321w64965862p66726481/explorer-table.plotKeys=%5B%5D&explorer-table.rowStart=0&explorer-table.rowCount=100&_r.drilldown=analytics.eventCategory:Wiki,analytics.eventAction:MainNav&explorer-graphOptions.selected=analytics.nthMonth

**Comment by peterbe** [comment]
"Get Firefox Help" and "Get web development help" and "Join the MDN Community" are not technical **feedback**. 
Possible remedies for that:

* Rename it from "Feedback" to "Feedback & Help" (or "Feedback/Help")
* Remove things that are not technically about feedback.

**Comment by HerminaC** [comment]
My feedback in line
    Send feedback (Already a duplicate link in the site footer) - _This is actually sending to a page explaining plenty of things, not only how to give feedback. Also, it is heavily used, so I would rather change it's name so that is more explicit rather than remove it @chrisdavidmills what do you think?._
    Get Firefox help (this could be a link in the site footer above the link to Firefox) - _agree let's take it out_
    Get Web Development help (links to Stackoverflow - Does anybody ever click this? Btw. we have amazing web development help one top-level menu to the left 😄) - _why do we even send to StackOverflow?? Should send to our forum._ 
    Join the community (perhaps replace Feedback with just this link at the top level. This seems in line with our renewed focus on community i.e. have this exposed at the top level)  - _this is a subset of the Send feedback page. Would take it out and maybe rename the thirst link with smth community related?@chrisdavidmills?_
    Report a content problem (Taken care of by the new Github section)  - _agree and also very low usage let's take it out_
    Report an issue (Taken care of by the new Github section) - _agree and also very low usage let's take it out_

**Comment by mattiapontonio** [comment]
Regardless of where to place them, it can be useful to reduce them to a single word to better see what it is.

- [Stack Overflow]()
- [Sumo]()
- [Community]()
- [Issues]()

After that, everyone will spontaneously return to their own domain.

**Comment by peterbe** [comment]
@escattone and I chatted about this and suggest the following, in this time-order:

1. First review and merge https://github.com/mdn/yari/pull/1948 to tie us over for the launch
2. If time allows this week, make a PR that deletes "Get Firefox Help" and "Get web development help"
3. Start a fresh new issue about potentially overhauling the whole menu and potentially replacing it all with just 1 link to somewhere.
